spectrum/autism_prevalence_map/static/autism_prevalence_map/tailwind.config.js

```diff
@@ -0,0 +1,46 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    content: [
+        './ts/**/*.ts',
+        '../svg/*.svg',
+        '../../templates/**/*.html',
+    ],
+    theme: {
+        fontSize: {
+        },
+        extend: {
+            fontFamily: {
+                'sans': ['GothiaSans', 'sans-serif'],
+                'serif': ['GothiaSerif', 'serif'],
+                'serif-text': ['GothiaSerifText', 'serif'],
+            },
+            borderRadius: {
+            },
+            letterSpacing: {
+            },
+            lineHeight: {
+            },
+            borderWidth: {
+            },
+            screens: {
+            },
+            spacing: {
+            },
+            colors: {
+                'red': '#910E1C',
+                'navy': '#003049',
+                'tan': '#FEF9EE',
+                'med-navy': '#2E5367',
+                'light-navy-2': '#DFE8EC',
+                'dark-gray': '#2D2D2D',
+                'light-gray': '#F4F5F6 ',
+                'blue': '#0B6BC3',
```

Comment:
1. I'll reach out to Joel about that today; I did my best, but clearly missed something; he's also using different names than we do on the TT site
2. this is because I'm trying to match these names 1:1 to what we use on TT
3. again, I'm trying to match the TT site's setup as closely as I can for interoperability reasons, but perhaps the best way would be to better-organize both of them